Pages/Page 1_Intro TOC.md

```diff
@@ -0,0 +1,107 @@
+This is page 1: it will contain an intro and a TOC for the rest of the content.
+
+#ClojureBridge Track 1 
+
+##Overview
+This cirriculum will introduce you to Clojure by creating a web app called Chatter. We will teach you to use the command line in Terminal, make code changes in LightTable, and push those changes to GitHub so you can share your app with the world. 
+
+Each chapter below will work through different concepts of coding and provide Clojure-specific instructions to help you create your app. Review this page, then work your way through each chapeter in the table of contents. 
+
+### Table Of Contents For The Workshop
+* [Chapter 1: NAME]
+* [Chapter 2: NAME]
+* [Chapter 3: NAME]
+
+
+## The Chatter App Goals & Tools
+
+### Specific Goal of Workshop
+
+Create a web app using Clojure code.
+
+We will introduce you to Clojure by creating and deplyoing a web app where you and your friends can post messages to each other.
+
+<<<_insert image of app_>>>
+
+### Additional Benefits & Learning Opportunities
+
+* Understand HTTP and web basics
+* Basic Clojure syntax
+* REPL <<<_(what does acronym stand for?)_>>> for fun and profit
+* Leiningen for project automation
+* Git for version control (branches, merges, commits)
+* Heroku deployment
+
+### Required Tools
+
+* Java 1.7+
+* Leiningen 2.5+
+* Git 2.1+
+* Firefox with Firebug
+* The Heroku Toolbelt
+
+_These should have all been installed at the Friday night Install Fest_
+
+
+## Readme
+This is the standard structure of most coding projects. You will have a Readme file that will provide a basic overview of the project and the programs you need to open the project. 
+
+### App
+
+In this curriculum we will create a web app, the Chatter app, that displays posted messages. 
+
+### Prerequisites
+
+You will need [Leiningen][] 2.0.0 or above installed. 
+
+_<<< is Leiningen = Terminal? >>>_
```

Comment:
Not quite, Leiningen is the build tool used to create and Clojure projects and manage their dependencies. It is a command line utility that runs in the terminal, but it is not the same as the terminal itself.